tests/performance/performancetests.sh

```diff
@@ -73,6 +73,10 @@ DATASET_SIZES=(497 4039 27770 334863 1134890 3774768 3072441)
 KNN4SOLS=(118 499 2805 1 64 1 1048907)
 KNN6SOLS=(357 547 5176 1 246 1 1453447)
 
+#the following datasets are used for benchmarking SpMV, SpMSpV, and SpMSpM
+MULTIPLICATION_DATASETS=(west0497.mtx nos3.mtx aft01.mtx G2_circuit.mtx hood.mtx apache2.mtx ecology2.mtx)
```

Comment:
What's the basis for having selected these datasets? The nos3, aft01, and hood matrices we typically have not used in performance benchmarks.

Before selecting though, maybe first to check: in normal operation (i.e., `make tests`), we will only run on the matrices that happen to be there, correct? (In contrast, for unit and smoke tests there are a limited number of matrices that are strongly recommended for testing purposes, which thus are included in a dataset downloading script in `./tools`).

Then if indeed assuming we just run on datasets that the user happens to have available, I think we should just use the union of datasets that are used for benchmarking in the following papers:
- https://dl.acm.org/doi/10.1145/3561652
- http://albert-jan.yzelman.net/PDFs/yzelman22-pp.pdf
- https://ieeexplore.ieee.org/document/6463397/

Could you make those changes?